src/components/SuggestionsList/SuggestionsListItem.css

```diff
@@ -6,6 +6,13 @@
   padding: 0.4rem 1.2rem 0.8rem 1.2rem;
 }
 
+.y-suggestion-list-item--badge {
+  background-color: $badgeColor__background;
+  fill: $badgeColor__fill;
```

Comment:
i don't think we should be enforcing this here, instead let the client set badgeContent in avatarProps for the item template